### DISCUSSION TITLE
i3 on Void autostart

### DISCUSSION THREAD

**Discussion by kora24** [i3/i3#6050] [Q&A]
Hi,

I have installed Void Base and added i3 but can't figure out how to make i3 start up automatically.

I have installed _xorg i3 i3status xinit lightdm lightdm-gtk3-greeter dmenu dbus_,
then performed `ln -s /etc/sv/dbus /var/service` and `ln -s /etc/sv/lightdm /var/service`,
and changed: `greeter-session=lightdm-gtk3-greeter`.

There is no _lightdm greeter_ on booting; _startx_ works but only when logged in as root.

Any thoughts, please?

Thank you.

**Comment by blacklightpy** [comment]
You should ask this in #voidlinux on https://web.libera.chat
